Lib/tokenize.py

```diff
@@ -96,7 +98,7 @@ def _all_string_prefixes():
                 result.add(''.join(u))
     return result
 
-@functools.lru_cache
+@functools.lru_cache(maxsize=_CACHE_MAX_SIZE)
```

Comment:
This mirrors https://github.com/python/cpython/blob/master/Lib/re.py#L288

Reply:
there's only ever going to be 1 (PseudoProg) + 4 (end string regexes) I believe in here

Reply:
:+1: Ok, then nevermind